From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Fri, 2 Mar 2012 02:52:36 +0100
Subject: Allow user_t and staff_t consolekit_dbus_chat() access so they can
 determine their session status - necessary to login in KDE
 sometimes.

---
 policy/modules/roles/staff.te      |    3 +++
 policy/modules/roles/unprivuser.te |    3 +++
 2 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/policy/modules/roles/staff.te b/policy/modules/roles/staff.te
index 2be17d2..4b9bffa 100644
--- a/policy/modules/roles/staff.te
+++ b/policy/modules/roles/staff.te
@@ -41,6 +41,9 @@ optional_policy(`
 optional_policy(`
 	sudo_role_template(staff, staff_r, staff_t)
 ')
+optional_policy(`
+	consolekit_dbus_chat(staff_t)
+')
 
 optional_policy(`
 	sysadm_role_change(staff_r)
diff --git a/policy/modules/roles/unprivuser.te b/policy/modules/roles/unprivuser.te
index baaba0f..32e5211 100644
--- a/policy/modules/roles/unprivuser.te
+++ b/policy/modules/roles/unprivuser.te
@@ -27,6 +27,9 @@ optional_policy(`
 optional_policy(`
 	xserver_role(user_r, user_t)
 ')
+optional_policy(`
+	consolekit_dbus_chat(user_t)
+')
 
 ifndef(`distro_redhat',`
 	optional_policy(`
